metrics/metrics.go

```diff
@@ -13,14 +15,17 @@ const (
 	metricNewVersion = "new_version"
 )
 
-func reportMetric(metricType string) {
-	token := util.GetEnv("METRICS_TOKEN")
-	body := strings.NewReader("")
-	_, err := http.Post(fmt.Sprintf("%s/%s?token=%s", baseURL, metricType, token), "text/plain", body)
-	util.Check(err)
+func reportMetric(ctx context.Context, metricType string) {
+	if token, ok := os.LookupEnv("METRICS_TOKEN"); ok {
+		body := strings.NewReader("")
+		_, err := http.Post(fmt.Sprintf("%s/%s?token=%s", baseURL, metricType, token), "text/plain", body)
+		util.Check(err)
+	} else {
+		util.Debugf(ctx, "ignoring metric report (env missing)")
```

Comment:
I wasn't imagining that we would log all the time, but that will make the debugging even more obvious. Is that fine @klausenbusk?

Reply:
Fine by me :)